Packaging/amiga/prep.sh

```diff
@@ -19,7 +21,7 @@ wget https://www.zlib.net/zlib-1.2.11.tar.gz -O zlib-1.2.11.tar.gz
 tar -xvf zlib-1.2.11.tar.gz
 mkdir -p zlib-1.2.11/build
 cd zlib-1.2.11/build
-cmake .. -DCMAKE_INSTALL_PREFIX=${SYSROOT} -DM68K_CPU=68040 -DM68K_FPU=hard -DM68K_COMMON="${M68K_COMMON} -O3 -fno-exceptions -w -noixemul -DBIG_ENDIAN -DAMIGA -fpermissive -std=c++14"
+cmake .. -DM68K_CPU=${M68K_CPU} -DM68K_FPU=${M68K_FPU} -DM68K_COMMON="${M68K_COMMON} -O3 -fno-exceptions -w -noixemul -DBIG_ENDIAN -DAMIGA -fpermissive -std=c++14"
```

Comment:
Quote standalone variables like this, no need for curly braces, double quotes just in case:

```
-DM68K_CPU="$M68K_CPU"
```